Rewrite landing navigation (Getting Started) (#1217)
prysmaticlabs/documentation: docs/backup-and-migration/switch-clients.md

This step is not required for nodes which are running on a virtual public cloud, but keep in mind - nodes will be required to run an execution client locally post merge!  

> By default, Prysm uses TCP/13000 and UDP/12000. Remove those two rules and replace them with the appropriate port forwarding for the client you are switching to. The process will be very similar to the steps laid out [here.](/manage-connections/configure-ports-and-firewalls.md#configure-your-router)

Reviewer: ```suggestion
By default, Prysm uses TCP/13000 and UDP/12000. Remove those two rules and replace them with the appropriate port forwarding for the client you are switching to. The process will be very similar to the steps on the [Configure your router page.](/manage-connections/configure-ports-and-firewalls.md#configure-your-router)
```
